src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1307,8 +1307,14 @@
           }
         },
 
+        "hostedViewerUri": {
+          "description": "An absolute URI at which the result can be viewed",
```

Comment:
>d [](start = 74, length = 1)

Add a period.

Reply:
Zapped 'em all.

---
In reply to: [229075914](https://github.com/Microsoft/sarif-sdk/pull/1122#discussion_r229075914) [](ancestors = 229075914)